### PR TITLE
Codesystemoid

### PR DESCRIPTION
**Pull request by mayerm94**
use codesystem oid instead of name

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/CYPRESS-519
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/helpers/results_helpers.js [hunk 1]
```diff
@@ -338,6 +338,20 @@ module.exports = class ResultsHelpers {
      * @returns {String} a pretty version of the given result
      */
   static prettyResult(result, indentLevel, keyIndent) {
+    const nameOidHash = {
```
Comment (jbradl11) [inline lib/helpers/results_helpers.js]: I think this list should match the list specified in the cqm-converter PR.
